### ISSUE TITLE
[BUG] iCAT flooded by metadata records in case of publication failure

### ISSUE THREAD

**Issue by ccacciari** [UtrechtUniversity/yoda-ansible#206]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

In a Yoda instance v1.8.4, we noticed thousands of AVUs for a data package stored in vault and in a state approved for publication:

```

AVUs defined for collection /yoda/home/vault-test/1_7test[1625056442]:
attribute: org_action_log
value: ["1630655106", "publication updated", "system"]
units:
----
attribute: org_action_log
value: ["1646934019", "publication updated", "system"]
units:
----
attribute: org_action_log
value: ["1645909519", "publication updated", "system"]
units:
----
```

The publication process was disrupted by a network issue: the port 22 was closed and the securecopy.sh script failed to perform the scp to copy the generated html page to the apache web server.
When we fixed the network problem and the publication process started to work again, the data package was not published because it was in state "unrecoverable". We changed the state to "retry" and then it was published.
However during the whole time that the network issue was affecting the system, the publication process kept creating new AVUs, for days, weeks, every minute, flooding the iCAT.

### Expected Behavior

We would expect that Yoda stops to add AVUs after a certain time.

### Steps To Reproduce

1. prevent the securecopy.sh script to work properly
2. approve a package for publication
3. monitor the metadata of the package in vault

### Environment

```markdown
- Yoda: 1.8.4
- Ansible:
- Operating System: CentOS 7
- Browser:
```

**Comment by lwesterhof** [comment]
Thanks for the report. We're now tracking it as ticket YDA-5036.

**Comment by lwesterhof** [comment]
Fixed in [610f56da98ff912ddf91a0472c0337b0f26e7647](https://github.com/UtrechtUniversity/yoda-ruleset/commit/610f56da98ff912ddf91a0472c0337b0f26e7647)
